gui/web/src/kelp-ops-api/sendMetricEvent.js

```diff
@@ -0,0 +1,8 @@
+export default (baseUrl, eventData) => {
```

Comment:
would it be a better API if this took `name` and `eventData` as two explicit fields rather than combining into one as `eventData`?

`name` is required, but `eventData` is optional -- this is how the backend data works (`sendMetricEventRequest` struct) in `send_metric_event.go` and should be reflected in the frontend API function.

this could be formatted as the following json object:
```
{
    name: "someName",
    data: { ... }
}
```